src/Popper.js

```diff
@@ -59,7 +59,7 @@ export class InnerPopper extends React.Component<PopperProps, PopperState> {
   static defaultProps = {
     placement: 'bottom',
     eventsEnabled: true,
-    referenceElement: undefined,
```

Comment:
To be the same value as it can be set by `ref` setter in React, there we have `(HTMLElement | null) => void;` it can't be undefined. Type in props is `referenceElement?: ReferenceElement,` so if the `referenceElement` is void it will be type `ReferenceElement` so `ReferenceObject | HTMLElement | null` that's way changed it to `null`

Reply:
In flow `?Type` can be Type, null or void. Please keep it as it was before

Reply:
Yep i know, as mention in most cases the values will be never void that's way this more explicit change. Will revert 👍